test/trajectory-subsetter/utilities.py

```diff
@@ -7,28 +7,26 @@
 
 from harmony import Client, Request
 from harmony.harmony import ProcessingFailedException
-import xarray as xr
+import datatree
 
 
 def compare_results_to_reference_file(results_file_name: str,
-                                      reference_file_name: str,
-                                      group_path: str) -> None:
-    """ Use native `xarray` functionality to compare data values and metadata
-        attributes of downloaded results to a reference file.
+                                      reference_file_name: str) -> None:
+    """ Use `DataTree` functionality to compare data values, variables, 
+        coordinates, metadata, and all their corresponding attributes of
+        downloaded results to a reference file.
 
     """
-    reference_data =  xr.open_dataset(reference_file_name, group=group_path)
-    results_data = xr.open_dataset(results_file_name, group=group_path)
+    reference_data =  datatree.open_datatree(reference_file_name)
+    results_data = datatree.open_datatree(results_file_name)
 
-    assert results_data.equals(reference_data), ('Output and reference files '
-                                                 'do not match.')
+    assert results_data.identical(reference_data), ('Output and reference files '
+                                                    'do not match.')
 
-    reference_data.close()
-    results_data.close()
     reference_data = None
     results_data = None
-
-
+    
+    
```

Comment:
Looks like some wild/unintended whitespace here also end of line 15, and maybe on L24?